met-web/src/components/comments/admin/review/emailPreview/EmailPreview.tsx

```diff
@@ -17,7 +17,7 @@ export default function EmailPreview({
     children: React.ReactNode;
     [prop: string]: unknown;
 }) {
-    const scheduledDate = formatDate(survey.engagement?.scheduled_date || '', 'MMM DD YYYY');
```

Comment:
The issue here is that the formatDate function is using the formatToPacificTimeZone function, which converts the date to the Pacific time zone before formatting it. This can cause a difference of one day if the date is near the end of the day in the Pacific time zone.

For example when i tested , 2024-05-10 is shown as April 09, 2024.
So using the same way in engagement view page using dayjs

Reply:
Check this https://github.com/bcgov/met-public/blob/main/met-api/templates/email_rejected_comment.html

This is the email that is generated when a comment is rejected, the component above is to preview it
![image](https://github.com/bcgov/met-public/assets/91914654/e29b097d-7680-48d0-a2b3-ddaa16992c69)

Reply:
![image](https://github.com/bcgov/met-public/assets/91914654/9aba9b82-9917-4b84-8e5d-a1ab022929e5)
![image](https://github.com/bcgov/met-public/assets/91914654/8b318fdb-9bfe-4c9d-af5f-7e5244b12a31)

Reply:
But the preview was using the engagement.scheduled_date instead